locations/spiders/starbucks_mena.py

```
@@ -1,65 +1,33 @@
+        "https://locations.starbucks.qa/robots.txt",
+        "https://locations.starbucks.sa/robots.txt",
+        "https://locations.starbucks.eg/robots.txt",
+        "https://locations.starbucks.com.bh/robots.txt",
+        "https://locations.starbucks.com.kw/robots.txt",
+        "https://locations.starbucks.com.lb/robots.txt",
+        "https://locations.starbucks.com.om/robots.txt",
+    ]
+    sitemap_rules = [
+        (r"https://locations.starbucks.com.\w+/[a-z-]+/[a-z-]+$", "parse_sd"),
+        (r"https://locations.starbucks.\w+/[a-z-]+/[a-z-]+$", "parse_sd"),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)